### PR TITLE
Feature#21 employee table

### PR DESCRIPTION
**Pull request by DiegoMoralesBravo**
# Description

What did you implemented/Why did you implemented this:

- Employee model created
- Table with all the register from the employees table

## Testing

Execute on terminal the following commands
> rails db:drop
> rails db:create
> rails db:migrate
> rails db:seed
> rails server

then navigate to /employees route

## Screenshots

image

### CODE REVIEW

**Review by AlanGonzalezM — APPROVED** [review]

**Review by Yaxkin121 — APPROVED** [review]
lgtm

File: app/views/employee/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      </div>
+    </section>
+    <% @employees.each do |employee| %>
+      <section class="employees-table">
+        <ul class= "employees-table-list" >
+          <li class="eployees-table-cell">
+            <p><%= employee.email %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.name %>Jose Juan Morales Bravo</p>
```
Comment (juperameza) [inline app/views/employee/index.html.erb:42]: Why is it fill with hard coded text?

File: test/models/employee_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class EmployeeTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (juperameza) [inline test/models/employee_test.rb:6]: If you don't need it delete it

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/employee/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          </li>
+          <li class="header-table-text">
+            <p>Private Number</p>
+          </li>
+          <li class="header-table-text">
+            <p>Actions</p>
+          </li>
+        </ul>
+      </div>
+    </section>
```
Comment (JuanequeX) [inline app/views/employee/index.html.erb:34]: If you have a `section` label, what is the reason for containing the 'ul' in a 'div'?

File: app/views/employee/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            <p><%= employee.employee_number %>303040394</p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.private_number %>232342354</p>
+          </li>
+          <li class="buttons-container">
+            <section>
+              <button class="button-edit">Edit</button>
+              <button class="button-delete">Delete</button>
+            </section>
```
Comment (JuanequeX) [inline app/views/employee/index.html.erb:57]: this not made seem to me, i think is better a `div` label here

File: co-autores.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Co-authored-by: DiegoMoralesBravo <diegomorales1359@gmail.com>
+Co-authored-by: ikerismak <ikerismak@gmail.com>
+Co-authored-by: EverardoAlcarazBaltazar <ealcaraz12@ucol.mx>
```
Comment (JuanequeX) [inline co-autores.txt:3]: it is totally necessary to have it inside of your project?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/employee/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.position %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.employee_number %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.private_number %></p>
+          </li>
```
Comment (JuanequeX) [inline app/views/employee/index.html.erb:52]: All these classes are named how `eployees-table-cell` fix them for `employees-table-cell`

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: test/models/employee_test.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+require "test_helper"
+
+class EmployeeTest < ActiveSupport::TestCase
+end
```
Comment (JuanequeX) [inline test/models/employee_test.rb:4]: I think `EmployeeTest` isn't the best name for a model and folder, what do you think?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: test/models/employee_test.rb (hunk 7 above)
Comment (ikerismak) [reply test/models/employee_test.rb:4]: those are a autogenerate files when run rails generate model, are they wrong?

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by dagp-ucol — APPROVED** [review]
LGTM

File: app/views/employee/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            <p><%= employee.employee_number %></p>
+          </li>
+          <li class="employees-table-cell">
+            <p><%= employee.private_number %></p>
+          </li>
+          <li class="buttons-container">
+            <section>
+              <button class="button-edit">Edit</button>
+              <button class="button-delete">Delete</button>
+            </section>
```
Comment (JuanequeX) [inline app/views/employee/index.html.erb:56]: please, use a `div` tag here

File: co-autores.txt (hunk 5 above)
Comment (JuanequeX) [reply co-autores.txt:3]: remove this file of the project.

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/employee/index.html.erb (hunk 8 above)
Comment (DiegoMoralesBravo) [reply app/views/employee/index.html.erb]: done

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**